assets/onboarding/use-fullscreen.jsx

```diff
@@ -1,18 +1,27 @@
-import { useEffect } from '@wordpress/element';
+import { useLayoutEffect } from '@wordpress/element';
 
-import './fullscreen.scss';
+/**
+ * Apply fullscreen view by hiding wp-admin elements via CSS.
+ *
+ * Allows setting additional classes on the body element.
+ * Fullscreen and classes are added when the component is mounted, and removed when unmounted.
+ *
+ * @param {string[]} bodyClasses Additional classes to be set.
+ */
+export function useFullScreen( bodyClasses = [] ) {
+	bodyClasses.push( 'sensei-wp-admin-fullscreen' );
```

Comment:
I think we could keep the attribute immutable here. Maybe something like `const classes = [ ...bodyClasses, 'sensei-wp-admin-fullscreen' ]`

Reply:
Yup, 9c4ecdc.